normalization/preservation-office-pdfa.py

```diff
@@ -0,0 +1,41 @@
+import subprocess
+import sys
+
+# Python 2/3 compatibility
+try:
+    range = xrange
+except NameError:
+    pass
+
+MAX_TRIES = 3
+
+
+def convert(document, target, max_tries=MAX_TRIES):
+    """
+    Attempt to convert a document into a PDF at the target location.
+
+    This continues up to max_tries times; it's possible for initial calls
+    to fail due to delays in spinning up a LibreOffice server, where
+    retries may succeed.
+    """
+    for _ in range(0, max_tries):
+        try:
+            # SelectPdfVersion=1 converts to PDF/A-1a, instead of
+            # the default PDF 1.4
+            subprocess.check_call(['unoconv', '-eSelectPdfVersion=1',
+                                   '--output={}'.format(target), document])
```

Comment:
This can be `['unoconv', '-eSelectPdfVersion=1', '--output', target, document]` and leave out the .format